common/src/main/java/com/wynntils/features/map/MainMapFeature.java

```diff
@@ -162,32 +156,25 @@ public void onScreenOpened(ScreenOpenedEvent.Post event) {
             return;
         }
 
-        LootChestType chestType = Models.LootChest.getChestType(event.getScreen());
+        LootChestTier chestType = Models.LootChest.getChestType(event.getScreen());
         if (chestType == null) return;
 
         if (chestType.ordinal() < minTierForAutoWaypoint.get().ordinal()) {
             lastChestPos = null;
             return;
         }
 
-        PoiLocation location = new PoiLocation(lastChestPos.getX(), lastChestPos.getY(), lastChestPos.getZ());
-        CustomPoi newPoi = new CustomPoi(
-                location,
-                chestType.getWaypointName(),
-                CommonColors.WHITE,
-                chestType.getWaypointTexture(),
-                CustomPoi.Visibility.DEFAULT);
+        Location location = new Location(lastChestPos.getX(), lastChestPos.getY(), lastChestPos.getZ());
```

Comment:
I'm not sure what type lastChestPos is, but can't it implement Location? Or have a getLocation()?

Reply:
It's a Minecraft BlockPos. Can't add any implementations (without mixins, but we don't want to do that).

Reply:
Maybe make a new constructor to Location that takes in a BlockPos then? (Maybe there already is one..?)